ui/views.py

```diff
@@ -182,6 +211,18 @@ def get(self, request, *args, **kwargs):
         return conditional_response(self, ttv_videos[0].video, *args, **kwargs)
 
 
+class TechTVDownload(VideoDownload):
+    """
+    Public video download for a TechTV-based URL
+    """
+    def get(self, request, *args, **kwargs):
+        ttv_videos = get_list_or_404(
```

Comment:
If there are two items in this list does it matter which one we pick?

Reply:
No

Reply:
All videos in the list should be identical (same S3 files copied over to multiple OVS `Video` objects).